index.html

```diff
@@ -486,10 +415,17 @@ <h6 class="code-block-title">Code Block</h6>
       </div>
     </footer>
 
+    <!-- Load AJS -->
+    <script src="./segment.js"></script>
+
+    <script src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@11.8.0/build/highlight.min.js"></script>
+
     <!-- Bootstrap JS -->
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js"></script>
 
     <!-- Custom JS -->
+    <script src="constants.js"></script>
+    <script src="helpers.js"></script>
```

Comment:
as long as the these 2 new files come before the `script.js` tag, the code in `script.js` has access to the variables